address-space-controller/src/main/resources/templates/brokered-space-infra.yaml

```diff
@@ -15,6 +15,21 @@ objects:
       app: enmasse
       infraUuid: ${INFRA_UUID}
       infraType: brokered
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: broker-support-${INFRA_UUID}
+    annotations:
+      addressSpace: ${ADDRESS_SPACE}
+    labels:
+      app: enmasse
+      infraType: standard
+      infraUuid: ${INFRA_UUID}
+      addressspace:
+  type: Opaque
+  data:
+    username: YnJva2VyLXN1cHBvcnQ=
```

Comment:
I think we said we wanted the support user id to be:

* unpredictable, but obviously a 'support' user id so its use is obvious from the logs.
* unlikely to collide with user id from the user domain

How about: `broker-support-<random>`